File: 9 Dec Remove Array Duplicates/code.cpp
class Solution {
  public:
    vector<int> findDuplicates(vector<int>& arr) {
        // code here
        set<int>s;
        vector<int>res;
        for(int i:arr){
            if(s.count(i))res.push_back(i);
            s.insert(i);
        }
        return res;
    }
};

File: 10 Dec Missing and Repeating/code.py
class Solution:
    def findTwoElement(self, arr):
        d=set(arr)
        a,b=-1,-1
        for i in range(1,len(arr)+1):
            if(i not in d):a=i
        d=set()
        for i in arr:
            if(i in d):
                b=i
            d.add(i)
        return [b,a]

